postgres/values.yaml

```diff
@@ -13,5 +13,6 @@ password: DEFINED-IN-REGION-CHART
 port_public: '5432'
 port_private: '5432'
 port_metrics: '9113'
+hba_cidr: 100.0.0.0/8
```

Comment:
I would vote to use 100.64.0.0/10, since 100.0.0.0/8 actually covers real IP's the Private space is only /12